src/applications/vaos/tests/e2e/vaos-helpers.js

```diff
@@ -284,6 +284,23 @@ function initAppointmentListMock(token) {
       },
     },
   });
+  mock(token, {
+    path: '/v0/vaos/community_care/eligibility/supported_sites',
+    verb: 'get',
+    query: 'site_codes[]=983',
+    value: {
+      data: [
+        {
+          id: '983',
+          type: 'object_type',
+          attributes: {
+            name: 'Cheyenne',
+            timezone: 'US/Mountain',
+          },
+        },
+      ],
+    },
+  });
```

Comment:
@jbalboni I haven't worked much with e2e tests yet.  Is this what you had in mind, or do I need to do more?

Reply:
Yep, this is what I mean.